beacon-chain/core/light-client/lightclient.go

```diff
@@ -194,7 +203,7 @@ func NewLightClientUpdateFromBeaconState(
 		// if finalized_block.message.slot != GENESIS_SLOT
 		if finalizedBlock.Block().Slot() != 0 {
 			// update.finalized_header = block_to_light_client_header(finalized_block)
-			finalizedLightClientHeader, err := BlockToLightClientHeader(attestedBlock.Version(), finalizedBlock)
+			finalizedLightClientHeader, err := BlockToLightClientHeader(ctx, currentSlot, finalizedBlock)
```

Comment:
I am still curious to see if anything bad happens when you call this function with a Capella slot on an Altair/Bellatrix block.

Reply:
> I am still curious to see if anything bad happens when you call this function with a Capella slot on an Altair/Bellatrix block.

Is that possible? A slot can only have a single block or no block at all, shouldn't their versions stay the same?